app/controllers/state_file/questions/unemployment_controller.rb

```diff
@@ -83,6 +83,9 @@ def state_file1099_params
           :federal_income_tax_withheld,
           :state_income_tax_withheld,
           :unemployment_compensation,
+          :federal_income_tax_withheld_amount,
+          :state_income_tax_withheld_amount,
+          :unemployment_compensation_amount,
```

Comment:
is this necessary given that you have the `sync_amount_columns` on the model before save?